### PR TITLE
BZ1987058: updated text for CD-ROM boot source selection and added explanations for the Advanced Storage settings

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with https://bugzilla.redhat.com/show_bug.cgi?id=1987058 and https://issues.redhat.com/browse/CNV-13249.

### CODE REVIEW

**Comment by lmandavi** [comment]
HTML content for the updated topic modules:

https://deploy-preview-35202--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html

https://deploy-preview-35202--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html#virt-adding-a-boot-source-web_virt-creating-vm-template

**Comment by lmandavi** [comment]
@yaacov Requesting SME review.

Update 1
Updated the order of the content for the paragraphs in the topic so that the flow of information is sequential and aligns with the recommendation in the defect: 

About virtual machines and boot sources:

https://deploy-preview-35202--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html

Update 2
Inserted new information in Step 5 for this topic:  https://deploy-preview-35202--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html#virt-adding-a-boot-source-web_virt-creating-vm-template

An additional disk is required for installing the operating system. This additional disk is added automatically and you can customize it later when you are creating the virtual machine. Click Mount this as a CD-ROM disk if you want this additional disk to be mounted as a CD-ROM.

This paragraph provides the following information: 1) the additional disk is added automatically 2) it can be customized later 3) specifically what happens when you click the checkbox for: Mount this as a CD-ROM disk 

Notes:
The Advanced Storage Settings were already explained in Step 6, 6a, 6b, 6c, 6d.

File: modules/virt-adding-a-boot-source-web.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ There are four methods for selecting and adding a boot source in the web console
 
 . In the *Add boot source to template* window, click *Select boot source*, select a method for creating a persistent volume claim (PVC): *Upload local file*, *Import via URL*, *Clone existing PVC*, or *Import via Registry*.
 
-.. Optional: Click *Mount this as a CD-ROM boot source* to add an additional disk and mount the disk as a CD-ROM.
+. An additional disk is required for installing the operating system. This additional disk is added automatically and you can customize it later when you are creating the virtual machine. Click *Mount this as a CD-ROM disk* if you want this additional disk to be mounted as a CD-ROM.
```
Comment (yaacov) [inline modules/virt-adding-a-boot-source-web.adoc:38]:
```suggestion
. Optional: Click *Mount this as a CD-ROM boot source* to add an additional disk and mount the disk as a CD-ROM. When mounting a CD-ROM an additional empty disk is added automatically, if you do not need an additinal empty disk when mounting a CD-ROM you can remove later when you create the virtual machine.
```

Usually users will use the CD-ROM to install an operating system on an empty disk, and then remove the CD ROM and use only the disk, If a user mount the boot source as a CD-ROM we assume they want to install an operating system to an empty disk from the CD-ROM , so we automatically add an empty disk mounted as DISK in addition to the CD-ROM that is mounted as CD-ROM

Comment (lmandavi) [reply modules/virt-adding-a-boot-source-web.adoc]: @yaacov I've reworded for simplicity and clarity. Please review. Thanks!

**Comment by lmandavi** [comment]
https://deploy-preview-35202--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html#virt-adding-a-boot-source-web_virt-creating-vm-template

**Review by yaacov — APPROVED** [review]

**Comment by yaacov** [comment]
LGTM

**Comment by lmandavi** [comment]
zpeng@redhat.com added the LGTM to the Jira story. 
Requesting peer review.

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

**Pull request merged**